config/project/entryTypes/externalContributor--6fd87453-1bda-4bb9-8c8d-4dd36185e75d.yaml

```diff
@@ -208,17 +202,17 @@ fieldLayouts:
             required: false
             tip: null
             type: craft\fieldlayoutelements\CustomField
-            uid: 86d64925-8b41-4086-9b09-e9390242dac8
+            uid: a31d5835-55ca-4d04-a986-223a2eb91cdd
             userCondition: null
             warning: null
             width: 100
         name: 'Sharing & SEO'
-        uid: 517184c5-5586-46c3-bb0b-8d65fb83345d
+        uid: 0b8b9169-bf5b-416d-8cce-6770df63ea80
         userCondition: null
-handle: person
+handle: externalContributor
```

Comment:
Should we use a common prefix or suffix for all People entry types? So `externalPerson` and `orgPerson`, or something similar.